### PR TITLE
filtering & unsaving from liked playlist

### LINKED ISSUE #103: Removing song from liked playlist (Estimate: 3 hr, 2 hr)

### PR DESCRIPTION
**Pull request by adellevo**

### CODE REVIEW

File: frontend/src/components/Post/Post.js [hunk 1]
```diff
@@ -196,20 +196,32 @@ function Post({
     }
   }
 
-  async function spotifyLike() {
+  async function spotifyLikeHandler() {
```
Comment (matthewhung09) [inline frontend/src/components/Post/Post.js:199]: refactor into 2 functions

File: frontend/src/components/Post/Post.js [hunk 2]
```diff
@@ -126,8 +126,8 @@ function Post({
         uris: [uri],
         // position: 0,
       };
-      console.log("data: ", data);
-      console.log("id: ", id);
+      // console.log("data: ", data);
+      // console.log("id: ", id);
```
Comment (gscotti6950) [inline frontend/src/components/Post/Post.js:130]: get rid of console logs

**Review by matthewhung09 — CHANGES_REQUESTED** [review]
other than the refactor lgtm

**Review by gscotti6950 — COMMENTED** [review]
Everything looks good. I would consider refactoring spotifyLikeHandler into an unlike and like function. Also get rid of the console logs. Otherwise, great work! :-)

File: frontend/src/components/Post/Post.js (hunk 1 above)
Comment (adellevo) [reply frontend/src/components/Post/Post.js:199]: Good suggestion, ended up splitting it.

File: frontend/src/components/Post/Post.js (hunk 2 above)
Comment (adellevo) [reply frontend/src/components/Post/Post.js:130]: Totally forgot about those, but they should all be gone now.

**Comment by adellevo** [comment]
Fixed issues in another branch (this one was very behind on CI/CD changes)

**Pull request closed without merging**